http3/conn.go

```diff
@@ -109,12 +126,24 @@ func (c *connection) acceptStream(ctx context.Context) (quic.Stream, *datagramme
 		strID := str.StreamID()
 		c.streamMx.Lock()
 		c.streams[strID] = datagrams
+		if c.idleTimeout > 0 {
+			if len(c.streams) == 1 {
```

Comment:
I don't understand the logic here.

Reply:
We stop the idle timer as soon as the pool is no longer empty (first stream added to the pool).